pkg/controllers/dynakube/extension/eec/reconciler_test.go

```diff
@@ -659,7 +659,7 @@ func TestVolumes(t *testing.T) {
 				Name: httpsCertVolumeName,
 				VolumeSource: corev1.VolumeSource{
 					Secret: &corev1.SecretVolumeSource{
-						SecretName: extensionsControllerTlsSecretName,
+						SecretName: dk.Name + consts.ExtensionsTlsSecretSuffix,
```

Comment:
are you intentionally using here string concatination?

Reply:
Thanks, I missed updating this to the new function :D